.github/workflows/github-actions.yaml

```
@@ -0,0 +1,22 @@
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues and NEW pull requests to project 1
+      uses: simonv3/assign-one-project-github-action@1.3.1
```

This should be the original name of the action 
```suggestion
      uses: srggrs/assign-one-project-github-action@1.2.1
```